src/main/java/io/github/debuggyteam/tablesaw/TableSawResourceLoader.java

```diff
@@ -1,24 +1,7 @@
 package io.github.debuggyteam.tablesaw;
 
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
-import javax.annotation.Nullable;
-
-import org.jetbrains.annotations.NotNull;
-import org.quiltmc.loader.api.QuiltLoader;
-import org.quiltmc.qsl.resource.loader.api.reloader.SimpleSynchronousResourceReloader;
-
-import com.google.gson.JsonArray;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
-import com.google.gson.JsonPrimitive;
+import com.google.gson.*;
```

Comment:
Star import intended?

Reply:
Probably not, I think intelliJ did it when I used ctrl+alt+o to optimize imports.

Reply:
That's probably intellij's default autoformat settings, which change 3+ similar imports to that (or was it 4+ ...?)

Reply:
3-5 imports get turned into star imports by default. My solution to that was this: 
![image](https://user-images.githubusercontent.com/17211100/226204721-9be9ccfe-42b6-4a93-bb12-a17f1abdc63d.png)